### PR TITLE
PBS Usage Example

### PR DESCRIPTION
**Pull request by ax3l**
Add a PBS Usage example.

- [x] add to sphinx docs (depends on #2005)

### CODE REVIEW

File: src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+PBS examples
+============
+
+Job Submission
+""""""""""""""
```
Comment (PrometheusPi) [inline src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst]: QUESTION:
In #2005 you removed double quotes and replaced them by single quotes.
Why do you use double quotes here?

File: src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+PIConGPU job submission on the *Hypnus* cluster at *HZDR*:
```
Comment (PrometheusPi) [inline src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst]: REQUIRED:
Typo: Hypnos

File: src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+Job Control
+"""""""""""
+
+* interactive job:
+
+  * ``qsub -I -q k20 -lwalltime=12:00:00 -lnodes=1:ppn=8``
+
+* details for my jobs:
+
+  * ``qstat -f 12345``
```
Comment (PrometheusPi) [inline src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst]: REQUIRED:
please add that `12345` is a PBS job id - or write `<job if>`

File: src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+* details for queues:
+
+  * ``qstat -a queueName`` show all jobs in a queue
+  * ``pbs_free -l`` compact view on free and busy nodes
+  * ``pbsnodes`` list all nodes and their detailed state (free, busy/job-exclusive, offline)
+
+* communicate with job:
+
+  * ``qdel 12345`` abort job
```
Comment (PrometheusPi) [inline src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst]: REQUIRED:
same as before

File: src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  * ``qsig -s Number 12345`` send signal or signal name to job
```
Comment (PrometheusPi) [inline src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst]: REQUIRED:either
```
qsig -s <(signal) number> <job id>`
# or 
qsig -s 1 12345 # with an explaining comment 
```

Comment (PrometheusPi) [inline src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst]: same below (but there the comments are mostly good)

File: src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst (hunk 1 above)
Comment (ax3l) [inline src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst]: Thanks, it should be `'''` here, too. (since both headlines are now on a lower section level)

**Comment by ax3l** [comment]
all updated and outdated syntax (wrong rst highlighting) in Slurm example fixed, too

**Pull request merged**